packages/components/src/Popover/Popover.mdx

```diff
@@ -0,0 +1,212 @@
+---
+name: Popover
+menu: Components
+route: /components/popover
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Content } from "../Content";
+import { Heading } from "../Heading";
+import { Button } from "../Button";
+import { Checkbox } from "../Checkbox";
+import { useState, useRef } from "react";
+import { Popover } from ".";
+
+# Popover
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Write a description about what Popover component is trying to achieve.
+
+```ts
+import { Popover } from "@jobber/components/Popover";
+```
+
+<Playground>
```

Comment:
I'm sort of thinking that it might be nicer to keep the first example here a bit more simple. Perhaps just something as simple as a button that shows a really simple popover?

Then we can get into more complex examples of open by default, etc lower down if we think we need to. 😄

Reply:
sounds good! 👍